source/helpers/render-semantic-release-commit-types.test.ts

```diff
@@ -0,0 +1,14 @@
+import {test, assert} from 'vitest';
+
+import {getSemanticCommitAndScope} from './render-semantic-release-commit-types.js';
+
+test('getSemanticCommitAndScope', () => {
+	assert.equal(getSemanticCommitAndScope('Commit message'), undefined);
+	assert.equal(getSemanticCommitAndScope('fix: Commit message'), ['fix']);
+	assert.equal(getSemanticCommitAndScope('feat: Commit message'), ['feat']);
+	assert.equal(getSemanticCommitAndScope('feat(scope): Commit message'), ['feat', 'scope']);
+	assert.equal(getSemanticCommitAndScope('feat(sco pe): Commit message'), ['feat', 'sco pe']);
+	assert.equal(getSemanticCommitAndScope('feat(): Commit message'), ['feat']);
+	assert.equal(getSemanticCommitAndScope('feat: Commit (message)'), ['feat']);
+	assert.equal(getSemanticCommitAndScope('fe at(scope): Commit message) '), [undefined, 'scope']);
```

Comment:
I’d expect this not to match anything. Perhaps the ChatGPT regex already addresses this

Reply:
Run `npm run` to see the available commands. You can run `npm run test` to run all the unit tests/lint/formatting checkers at once. If you want just the unit tests specifically, it’s better you run `npx vitest` because that enables the watcher and the tests are rerun automatically.